external/epsearchast/v3/mongo/mongo_query_builder.go

```diff
@@ -55,6 +55,12 @@ func (d DefaultMongoQueryBuilder) VisitLike(first, second string) (*bson.D, erro
 	return &bson.D{{first, bson.D{{"$regex", d.ProcessLikeWildcards(second)}}}}, nil
 }
 
+func (d DefaultMongoQueryBuilder) VisitIsNull(first string) (*bson.D, error) {
+	// https://www.mongodb.com/docs/manual/reference/operator/query/eq/#std-label-eq-usage-examples
+	// This is equivalent to { key: value } but makes for easier tests.
```

Comment:
```suggestion
	// https://www.mongodb.com/docs/manual/tutorial/query-for-null-fields/#equality-filter
	// This will match fields that either contain the item field whose value is nil or those that do not contain the field
	// Customize this method if you need different nil handling (i.e., explicit nil)	
```